src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -507,6 +519,52 @@ $pf-c-dropdown--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "
     }
   }
 
+  &.pf-m-secondary.pf-m-split-button:not(.pf-m-disabled) {
+    --pf-c-dropdown__toggle-button--Color: var(--pf-c-dropdown__toggle--m-secondary--m-split-button--Color);
+
+    &:hover,
+    &:focus,
+    &:active {
+      --pf-c-dropdown__toggle--m-secondary--hover--before--BorderWidth: var(--pf-c-dropdown__toggle--m-secondary--m-split-button--hover--before--BorderWidth);
+    }
+
+    > * {
+      &::before {
+        border: var(--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--BorderWidth) solid var(--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--BorderColor);
+      }
+
+      &:hover,
+      &:focus,
+      &:active {
```

Comment:
I wonder if we should have unique vars for these states - seems like we do with the other variations.